app/Policies/Paie/SalaryPaymentRunPolicy.php

```
@@ -0,0 +1,72 @@
+use Illuminate\Foundation\Auth\User as AuthUser;
+
+class SalaryPaymentRunPolicy
+{
+    use HandlesAuthorization;
+
+    public function viewAny(AuthUser $authUser): bool
+    {
+        return $authUser->can('ViewAny:SalaryPaymentRun');
+    }
+
+    public function view(AuthUser $authUser, SalaryPaymentRun $run): bool
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$run'. (undefined)

(UnusedFormalParameter)